flocker/provision/_ssh/_fabric.py

```diff
@@ -0,0 +1,68 @@
+from fabric.api import settings, run, put, sudo
+from fabric.network import disconnect_all
+from StringIO import StringIO
+
+from ._model import Run, Sudo, Put, Comment, RunRemotely
+
+from effect import (
+    sync_performer,
+    sync_perform,
+    TypeDispatcher, ComposedDispatcher,
+    )
+
+from .._effect import dispatcher as base_dispatcher
+
+
+@sync_performer
+def perform_run(dispatcher, intent):
+    run(intent.command),
+
+
+@sync_performer
+def perform_sudo(dispatcher, intent):
+    sudo(intent.command),
+
+
+@sync_performer
+def perform_put(dispatcher, intent):
+    put(StringIO(intent.content), intent.path),
+
+
+@sync_performer
+def perform_comment(disptacher, intent):
```

Comment:
`disptacher` > `dispatcher`. Is there something (like lambda syntax?) which could make it clear in the dispatcher that this does nothing?